src/styles/block/nav.scss

```diff
@@ -0,0 +1,27 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+
+    margin: 0;
+    padding: 0;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    transition-duration: $transition-speed;
+    line-height: $header-height;
+
+    display: block;
+    color: $color-text;
+  }
+
+  &__item:not(:last-child) {
+    padding-right: 20px;
+  }
+
+  &__link:hover {
+    color: $header-color;
```

Comment:
Ensure that `$header-color` is the correct variable for the hover state of the link. If it's a color specific to the header, you might want to use a more generic variable for link hover states or define a new one if necessary.